docs/Editing/code-formatting.md

```diff
@@ -4,3 +4,19 @@ The extension provides code formatting for BrightScript and BrighterScript files
 1. Update your user or workspace settings with various `brightscript.format.*` options (see the [extension settings](../extension-settings.html) page for more info).
 
 2. Create a `bsfmt.json` file at the root of your project. See all of the available `bsfmt.json` options [here](https://github.com/rokucommunity/brighterscript-formatter#bsfmtjson-options). Please note, if a `bsfmt.json` file exists, all formatter-related user/workspace settings will be ignored. 
+
+## Editor word separator
+
+By default, VSCode uses `$`, `%`, `!`, `#` and `&` as word separators. Because of this, the full name of a strictly typed String 
+(resp. Integer, Float, Double and LongInteger) is not taken as a whole with selection and other word navigation feature. You can 
+edit the word separator value for Brightscript by adding the following setting:
+
+```
+{
+    "[brightscript]": {
```

Comment:
Can you add another section for `[brighterscript]` as well?

Reply:
Sure thing